crates/evm/src/executor/inspector/access_list.rs

```diff
@@ -73,23 +68,22 @@ impl<DB: Database> Inspector<DB> for AccessListTracer {
             opcode::BALANCE |
             opcode::SELFDESTRUCT => {
                 if let Ok(slot) = interpreter.stack().peek(0) {
-                    let addr: Address = H256::from_uint(&ru256_to_u256(slot)).into();
+                    let addr: Address = Address::from_slice(&slot.to_be_bytes::<32>()[12..]);
                     if !self.excluded.contains(&addr) {
                         self.access_list.entry(addr).or_default();
                     }
                 }
             }
             opcode::DELEGATECALL | opcode::CALL | opcode::STATICCALL | opcode::CALLCODE => {
                 if let Ok(slot) = interpreter.stack().peek(1) {
-                    let addr: Address = H256::from_uint(&ru256_to_u256(slot)).into();
+                    let addr: Address = Address::from_slice(&slot.to_be_bytes::<32>()[12..]);
```

Comment:
Probably better as `Address::from_word(slot.into())`

Reply:
nice, definitely more elegant

Reply:
done in https://github.com/foundry-rs/foundry/pull/5770/commits/a1fe075f764a79e636ab6237cc9ce2858ff9d31a

Reply:
missing this line, you changed only the one above